stan/math/prim/scal/prob/bernoulli_lccdf.hpp

```diff
@@ -20,6 +20,21 @@
 namespace stan {
   namespace math {
 
+    /**
+     * Log complementary CDF of the Bernoulli distribution. If vectors of matching lengths
+     * are supplied, returns the log sum of the probabilities.
```

Comment:
fair point. I think explaining the broadcast behavior in each doc block would be overdoing it, will reword across the board.